### DISCUSSION TITLE
How to see the server logs?

### DISCUSSION THREAD

**Discussion by ArminEskandarii** [ente-io/ente#4377] [Q&A]
Hi, I succeeded at as selfhosting Ente on a vps, now the problem is that i wanna get the email and it doesn't send me, So i accross this [guide](https://help.ente.io/self-hosting/faq/otp) that it's saying i should get  the code from server log or database.

Well how can I check the logs at the server?
my server OS is Ubuntu 24

**Comment by mngshm** [comment]
if docker `docker compose logs museum`. Or else if you have hosted it via systemd then do `sudo journalctl -fu museum.service` or whatever the service name is.
